packages/lib/actionClass/service.ts

```diff
@@ -49,34 +45,44 @@ export const getActionClasses = (environmentId: string, page?: number): Promise<
         throw new DatabaseError(`Database error when fetching actions for environment ${environmentId}`);
       }
     },
-    [`environments-${environmentId}-actionClasses`],
+    [`getActionClasses-${environmentId}-${page}`],
     {
-      tags: [getActionClassesCacheTag(environmentId)],
+      tags: [actionClassCache.tag.byEnvironmentId(environmentId)],
       revalidate: SERVICES_REVALIDATION_INTERVAL,
     }
   )();
 
-export const getActionClass = async (actionClassId: string): Promise<TActionClass | null> => {
-  validateInputs([actionClassId, ZId]);
-  try {
-    let actionClass = await prisma.eventClass.findUnique({
-      where: {
-        id: actionClassId,
-      },
-      select,
-    });
+export const getActionClass = async (actionClassId: string): Promise<TActionClass | null> =>
+  unstable_cache(
+    async () => {
+      validateInputs([actionClassId, ZId]);
 
-    return actionClass;
-  } catch (error) {
-    throw new DatabaseError(`Database error when fetching action`);
-  }
-};
+      try {
+        const actionClass = await prisma.eventClass.findUnique({
+          where: {
+            id: actionClassId,
+          },
+          select,
+        });
+
+        return actionClass;
+      } catch (error) {
+        throw new DatabaseError(`Database error when fetching action`);
+      }
+    },
+    [`getActionClass-${actionClassId}`],
+    {
+      tags: [actionClassCache.tag.byActionClassId(actionClassId)],
```

Comment:
do we want to name it `byActionClassId` or just `byId`? As it's the field `id` of the actionClass I think id makes more sense

Reply:
this makes sense